services/fileServices/MdPageServices/ContactUsPageService.js

```diff
@@ -0,0 +1,56 @@
+const {
+  retrieveDataFromMarkdown,
+  convertDataToMarkdown,
+} = require("@utils/markdown-utils")
+
+const CONTACT_US_FILE_NAME = "contact-us.md"
+const CONTACT_US_DIRECTORY_NAME = "pages"
+
+class ContactUsPageService {
+  constructor({ gitHubService, footerYmlService }) {
+    this.gitHubService = gitHubService
+    this.footerYmlService = footerYmlService
+  }
+
+  async read(reqDetails) {
+    const { content: rawContent, sha } = await this.gitHubService.read(
+      reqDetails,
+      {
+        fileName: CONTACT_US_FILE_NAME,
+        directoryName: CONTACT_US_DIRECTORY_NAME,
+      }
+    )
+    const { frontMatter, pageContent } = retrieveDataFromMarkdown(rawContent)
+    const { content: footerContent } = await this.footerYmlService.read(
+      reqDetails
+    )
+    frontMatter.feedback = footerContent.feedback
+    return { content: { frontMatter, pageBody: pageContent }, sha }
+  }
+
+  async update(reqDetails, { content, frontMatter, sha }) {
+    const newContent = convertDataToMarkdown(frontMatter, content)
+    const { newSha } = await this.gitHubService.update(reqDetails, {
+      fileContent: newContent,
+      sha,
+      fileName: CONTACT_US_FILE_NAME,
+      directoryName: CONTACT_US_DIRECTORY_NAME,
+    })
+    const {
+      content: footerContent,
+      sha: footerSha,
+    } = await this.footerYmlService.read(reqDetails)
+    footerContent.feedback = frontMatter.feedback
```

Comment:
on this point, is my understanding that we maintain the `feedback` property in `footerContent` and when something (here, `frontMatter`) requires it, we read from `footerContent` and write to `footerContent` on update?

Reply:
See above!

Reply:
same - add the explanation in the comment here too!

Reply:
https://github.com/isomerpages/isomercms-backend/pull/349/commits/3b6ca17dd13e63e6a283354987a50c0a4fd60c2d